### PR TITLE
[xcode12] Bump for beta 3

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by mandel-macaque** [comment]
I wonder what has happened with all the network API on the watch... so many unknown PInkove messages..

**Comment by spouliot** [comment]
> I wonder what has happened with all the network API on the watch... so many unknown PInkove messages..

@mandel-macaque the headers are broken so NetworkExtension is excluded from xtro on watchOS

**Comment by spouliot** [comment]
No iOS 32bits results (all timeouts) 😞 

```
      <SimDevice UDID="DA24561E-C03F-44DD-8883-BDF243A70D92" Name="iPhone-5 (iOS-10-3) - created by xharness">
        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-10-3</SimRuntime>
        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5</SimDeviceType>
        <DataPath>/Users/builder/Library/Developer/CoreSimulator/Devices/DA24561E-C03F-44DD-8883-BDF243A70D92</DataPath>
        <LogPath>/Users/builder/Library/Logs/CoreSimulator/DA24561E-C03F-44DD-8883-BDF243A70D92</LogPath>
      </SimDevice>
```

note the `10-3` and not `10.3` which is not identical to builds with beta 2...

After the XML there's no
```
Found simulator: *
```
entry for any 10.3 simulator.

That might be an `mlaunch` issue as it affects both VSfM and xharness. The simulator does start (and existing apps works) but the new app is never deployed (nor obviously executed).

**Comment by spouliot** [comment]
beta 2
```
     <SimDevice UDID="4F854F8B-5D9D-45EB-A839-6F3DE7849A4B" Name="iPhone 5 (iOS 10.3) - created by xharness">
        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-10-3</SimRuntime>
        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5</SimDeviceType>
        <DataPath>/Users/builder/Library/Developer/CoreSimulator/Devices/4F854F8B-5D9D-45EB-A839-6F3DE7849A4B</DataPath>
        <LogPath>/Users/builder/Library/Logs/CoreSimulator/4F854F8B-5D9D-45EB-A839-6F3DE7849A4B</LogPath>
      </SimDevice>
 ```

**Comment by spouliot** [comment]
beta 3
```
     <SimDevice UDID="DA24561E-C03F-44DD-8883-BDF243A70D92" Name="iPhone-5 (iOS-10-3) - created by xharness">
        <SimRuntime>com.apple.CoreSimulator.SimRuntime.iOS-10-3</SimRuntime>
        <SimDeviceType>com.apple.CoreSimulator.SimDeviceType.iPhone-5</SimDeviceType>
        <DataPath>/Users/builder/Library/Developer/CoreSimulator/Devices/DA24561E-C03F-44DD-8883-BDF243A70D92</DataPath>
        <LogPath>/Users/builder/Library/Logs/CoreSimulator/DA24561E-C03F-44DD-8883-BDF243A70D92</LogPath>
      </SimDevice>
```

File: Versions-mac.plist.in [hunk 1]
```diff
@@ -18,6 +18,7 @@
 			<string>10.14</string>
 			<string>10.15</string>
 			<string>10.16</string>
```
Comment (rolfbjarne) [inline Versions-mac.plist.in:20]: Do we want to remove `10.16` here?

File: tools/common/Frameworks.cs [hunk 2]
```diff
@@ -535,15 +535,8 @@ public static void Gather (Application app, AssemblyDefinition product_assembly,
 	// This checks if a framework is unavailable due to bugs in Xcode (such as Apple forgetting to ship a library or headers for a framework, which seems to happen at least once a year).
 	public static bool IsFrameworkBroken (Application app, string framework)
 	{
-		if (app.IsSimulatorBuild && Driver.XcodeProductVersion == "12A6163b" /* Xcode 12 beta 12 */) {
+		if (app.IsSimulatorBuild && Driver.XcodeProductVersion == "12A8169g" /* Xcode 12 beta 3 */) {
 			switch (framework) {
-			// Apple seems to have forgotten to ship the several libraries for the simulator in Xcode 12 betas (it's still available for device builds).
```
Comment (rolfbjarne) [inline tools/common/Frameworks.cs:540]: This particular line is still correct for the MediaSetup framework, right?

**Comment by rolfbjarne** [comment]
> The `Name` format is not identical (see next 2 comments) but that might not be related to the issue.

I don't think this is the issue.

Apple probably changed something in their CoreSimulator API.

**Comment by rolfbjarne** [comment]
> > The `Name` format is not identical (see next 2 comments) but that might not be related to the issue.
> 
> I don't think this is the issue.
> 
> Apple probably changed something in their CoreSimulator API.

Actually I can't get the "iPhone 5 / iOS 10.3" simulator to work in Xcode either, the same thing happens, it never installs/launches.

File: Versions-mac.plist.in (hunk 1 above)
Comment (spouliot) [reply Versions-mac.plist.in:20]: part of https://github.com/xamarin/xamarin-macios/issues/8943
My guess is yes, but I was not sure this bump was the best place to do it :)

**Comment by spouliot** [comment]
> it never installs/launches.

I had not tried yet from Xcode. We can launch the sim, but the app is not deployed :(

File: tools/common/Frameworks.cs (hunk 2 above)
Comment (spouliot) [reply tools/common/Frameworks.cs:540]: yes, I cut too large :)

**Comment by spouliot** [comment]
I filed the issue about Xcode and 32bits simulator here -> [FB8114432](https://github.com/xamarin/maccore/issues/2270)

**Comment by spouliot** [comment]
Failure is a known issue -> https://github.com/xamarin/maccore/issues/1028

**Pull request merged**